pandas/core/dtypes/concat.py

```diff
@@ -72,6 +72,8 @@ def concat_compat(to_concat, axis: int = 0):
     ----------
     to_concat : array of arrays
     axis : axis to provide concatenation
+    ea_compat_axis : bool, default False
+        For ExtensionArray compat, behave as if axis == 1
```

Comment:
Can you clarify what this means in practice to behave "as if axis=1"? Because I assume the arrays are still concatenated with axis=0?

Reply:
just matters for the dropping-empty check; will edit to clarify